datasets/za/mm_officials/za_mm_officials.yaml

```diff
@@ -32,10 +32,10 @@ data:
 assertions:
   min:
     schema_entities:
-      Person: 750
+      Person: 1500
   max:
     schema_entities:
-      Person: 800
```

Comment:
I wonder where this is going wrong.
If I visit https://municipaldata.treasury.gov.za/api/cubes/officials/facts I get
- 396 instances of `"Mayor`
- 292 instances of `"municipal manager`
- 292 instances of `"chief fina`

So I don't know how we're ending up at 2134 Persons

Reply:
for context, South Africa has 257 municipalities. Disestablished municipalities don't exist any more and their area has been taken over by new/existing municipalities. So even 292 is a lot of managers and CFOs